transit/tests/api_test/mixin/user_mixin.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import (
+    User,
+    Permission
+)
+from django.contrib.contenttypes.models import ContentType
+
+from transit.models import UserManager
+
+
+class UserMixin:
+    _TEST_USERNAME = "TestUser"
+    _TEST_PASSWORD = "TestPassword"
+    _TEST_DATA_USER = {
+        "username": _TEST_USERNAME,
+        "is_staff": True,
+        "password": _TEST_PASSWORD
+    }
+
+    def get_or_create_user(self):
+        user = User.objects.filter(username=self._TEST_USERNAME).first()
+        if user is None:
+            return self.__create_user()
+        return user
+
+    def add_permissions_to_manage_users(self):
+        user = User.objects.filter(username=self._TEST_USERNAME).first()
+        if user:
```

Comment:
you can either change `.first()` to `.get()` or raise an exception if user was not found. 
At this point we expects that user exists therefore we should give developer explicit information if there's no user.

Reply:
I 've just used 'get'